standalone/content/css/webapp.css

```diff
@@ -478,6 +573,84 @@ html[dir="rtl"] .standalone-info-bar > .media-control-buttons > .btn-hangup:afte
   color: #999;
 }
 
+@media screen and (max-width:1024px) {
```

Comment:
From testing Safari on my Mac, I think this wants to be somewhere around 1120px - 1024 makes content disappear off the sides.